understanding/20/contrast-enhanced.html

```diff
@@ -67,7 +67,10 @@ <h2>Intent of Contrast (Enhanced)</h2>
             resulting image is roughly equivalent to 1.2 and 1.5 em or to 120% or 150% of the
             default size for body text as rendered by the browser.
          </p>
-      	<p>The 7:1 and 4.5:1 contrast ratios referenced in this Success Criteria are intended to be treated as threshold values. When comparing the computed contrast ratio to the Success Criteria ratio the computed values should not be rounded (eg. 4.499:1 would not meet the 4.5:1 threshold).</p>
+         <p>The 7:1 and 4.5:1 contrast ratios referenced in this Success Criterion are intended to be
+            treated as threshold values. When comparing the computed contrast ratio to the Success Criterion
+            ratio, the computed values should not be rounded (e.g. 4.499:1 would not meet the 4.5:1 threshold).</p>
```

Comment:
it seems that after merging the main branch back into this (https://github.com/w3c/wcag/pull/2254#pullrequestreview-923310979), the only real change here and in the contrast minimum file is that it's split over multiple lines, and changes the old `Success Criteria` to `Success Criterion` (singular)